stream_alert/shared/config.py

```diff
@@ -33,6 +33,24 @@ class TopLevelConfigKeys(object):
 class ConfigError(Exception):
     """Exception class for config file errors"""
 
+class SchemaSorter(object):
+    """Sort Schema by Priority"""
+
+    # Set a default index to -1
+    max_index = -1
+
+    @classmethod
+    def sort(cls, key_and_value_tuple):
```

Comment:
I suggest coming up with a better function name than `sort`. This function and this class do not do sorting; They act more like a namespaced comparison function.

Reply:
This one in particular is a sort key function, so maybe it something like that? `sort_key`